test/bin/encore.js

```diff
@@ -39,6 +39,7 @@ describe('bin/encore.js', function() {
             `
 const Encore = require('../../index.js');
 Encore
+    .enableSingleRuntimeChunk()
```

Comment:
We could but that's not a big issue since the current version allows not to call `enableSingleRuntimeChunk()`/`disableSingleRuntimeChunk()` (unless you also called `createSharedEntry()` it defaults to the `disableSingleRuntimeChunk()` behavior with deprecation messages).

Reply:
I mean, we should avoid using deprecated API in our testsuite, unless we are specifically testing the BC layer (in which case the test gets removed when dropping it, or changed to a test asserting an exception).
That's the policy we have in Symfony itself (we even make the testsuite fail if deprecations are triggered in non-legacy tests).